osmo_jupyter/timezone.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+
+import pytz
+import pandas as pd
+
+
+OSMO_HQ_TIMEZONE = pytz.timezone('US/Pacific')
+SQL_TIME_FORMAT = '%Y-%m-%d %H:%M:%S'  # Time format favored by mysql
+
+
+def to_aware(local_time):
```

Comment:
especially since, as a non-dev, `to_aware` might not mean all that much as a function name.

Though I guess maybe the main usage of functions from this module will be `utc_series_to_local`?

Reply:
+1 to function name and usage ex

Reply:
yeah, this is kinda weird because the only function in here I really expect folks to be using is `utc_series_to_local`. I'm finally convinced - gonna move those other two functions into `db_access` and make them private.